packages/passport-interface/src/RequestTypes.ts

```diff
@@ -626,6 +626,9 @@ export interface FrogCryptoComputedUserState {
 }
 export interface FrogCryptoUserStateResponseValue {
   feeds: FrogCryptoComputedUserState[];
+  frogs: {
+    count: number;
+  };
```

Comment:
this data structure has had me confused this entire PR. suggest replacing this with 

```suggestion
  // quantity of different types of frogs that can spawn in the pcd-verse
  possibleFrogsCount: number;
```